lib/router.js

```diff
@@ -51,6 +51,10 @@ module.exports = (config) => {
     path: paths.translations + '/__lng__/__ns__.json'
   });
 
+  router.use(hof.middleware.deepTranslate({
+    translate: i18n.translate.bind(i18n)
+  }));
+
   router.use(mixins(fields, {
     translate: i18n.translate.bind(i18n)
```

Comment:
the deepTranslate middleware attaches a translate function to `req` which is used in `mixins` - this is overridden if you pass a custom translate function so this should be removed to take advantage